### ISSUE TITLE
Maybe a BUG: ins_label or label(bag based) for max select

### ISSUE THREAD

**Issue by jjyunlp** [thunlp/OpenNRE#85]
When use bag_maximum for select
The "self.ins_label" in this train_demo.py
```python
self._train_logit, train_repre = nrekit.network.selector.bag_maximum(x_train, self.scope, self.ins_label, self.rel_tot, True, keep_prob=0.5)
```
And next in selector.py as the "query"

```python
for i in range(scope.shape[0]):
                bag_hidden_mat = x[scope[i][0]:scope[i][1]]
                instance_logit = tf.nn.softmax(__logit__(bag_hidden_mat, rel_tot), -1) # (n', hidden_size) -> (n', rel_tot)
                j = tf.argmax(instance_logit[:, query[i]], output_type=tf.int32)
                bag_repre.append(bag_hidden_mat[j])
```

As the query[i] is a labled tag for get the max score in "right label" column during training, it should be the bag-based label.
If use the ins_label, an instance label from previous bag will be used for following bag since one bag may contains more than one instance.

**Comment by gaotianyu1350** [comment]
Yes, you're right. The bug has been fixed. Thanks!
